mitodl/ol-data-platform
# add user_hashed_id to combined marts

### What are the relevant tickets?




https://github.com/mitodl/hq/issues/5715

### Description (What does it do?)

Adding `user_hashed_id` and unique test to marts__combined__users and other marts so that it can be used for joining. Without this column, it isn't easy to join marts__combined__users.


### How can this be tested?


dbt build -select 1+marts__combined__users --- the direct upstream model should be enough
dbt build -select marts.combined

You will get a failed [unique test ](https://github.com/mitodl/ol-data-platform/blob/main/src/ol_dbt/models/marts/combined/_marts__combined__models.yml#L435)on marts__combined_program_enrollment_detail, which is unrelated to this PR. https://github.com/mitodl/hq/issues/5756 is created to address that.
